.azure/scripts/osx-python.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+PYTHON_VERSION="$1"
+
+case $PYTHON_VERSION in
+3.6)
+  FULL_VERSION=3.6.8
+  ;;
+3.7)
+  FULL_VERSION=3.7.3
+  ;;
+3.8)
+  FULL_VERSION=3.8.2
+  ;;
+esac
+
+INSTALLER_NAME=python-$FULL_VERSION-macosx10.9.pkg
+URL=https://www.python.org/ftp/python/$FULL_VERSION/$INSTALLER_NAME
+
+PY_PREFIX=/Library/Frameworks/Python.framework/Versions
+
+set -e -x
+
+curl $URL > $INSTALLER_NAME
+
+sudo installer -pkg $INSTALLER_NAME -target /
+
+sudo rm /usr/local/bin/python
+sudo ln -s /usr/local/bin/python$PYTHON_VERSION /usr/local/bin/python
+
+which python
+python --version
+python -m ensurepip
+python -m pip install setuptools wheel
```

Comment:
have you considered using pyenv for setting the actual python version? https://github.com/pyenv/pyenv It should make our lives easier switching between interpreters, Symlinking seems a bit like brute force (and what would happen if /usr/bin/ etc. takes a higher priority in PATH?)

Reply:
The recipe for this was from the official azure blog on wheel releases.  I just copied.  There may be better methods.   The blog was also pretty old so I had to up versions to get it to work.